test/integration/aggregates/aggregates_integration_test.go

```diff
@@ -170,33 +178,30 @@ func testAggregatesCase(t *testing.T, testConfig *TestConfig) {
 				totalCount += int(v)
 			}
 
-			if tm > testConfig.testBaseTime && tm < testConfig.testStartTime-testConfig.queryStep {
-				if aggr == "count" && int64(v) != expectedCount {
-					fmt.Println("\n***", testConfig.testStartTime, testConfig.testStartTime-3*testConfig.queryStep-int64(time.Minute))
-					t.Errorf("Count is not ok - expected %d, got %f at time %d", expectedCount, v, tm)
-				}
-				if aggr == "sum" && v != expectedSum {
-					t.Errorf("Sum is not ok - expected %f, got %f at time %d", expectedSum, v, tm)
-				}
-				if aggr == "avg" && v != expectedSum/float64(expectedCount) {
-					t.Errorf("Avg is not ok - expected %f, got %f at time %d", expectedSum/float64(expectedCount), v, tm)
-				}
-			}
+			if tm > testConfig.testStartTime && tm < testConfig.testEndTime-testConfig.queryStep {
 
+				assert.Condition(t, func() bool { return aggr == "count" && int64(v) != expectedCount },
```

Comment:
The error message will be displayed whenever the condition it `false` looks like it is the opposite of what you want, isn't it?